docs/develop/01_sdk/01_quickstart.md

```diff
@@ -125,9 +124,9 @@ You should add all other code before this function call:
 
 <SnippetBlock
 className="language-ts"
->
-{Disconnect}
-</SnippetBlock>
+
+> {Disconnect}
```

Comment:
Something else changing these, there shouldn't be any changes needed here, the extra `>`s will break things

Reply:
Yes It was because of linter but it fixed in here: https://github.com/KILTprotocol/docs/pull/329/commits/2fb4bf229be1bd8c28ed45066790a66d8b5f1e29